esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -102,25 +102,30 @@ Troubleshooting:
 //**********************//
 //*** WiFi settings ***//
 
+#define USE_BROADCAST_FOR_UDP // Comment out if you want to use only the first WiFi client to connect
+
 // for TCP, UDP (only for these two)
 // ssid = "" results in a default name, like "mLRS-13427 AP UDP"
 // password = "" makes it an open AP
 String ssid = ""; // "mLRS AP"; // Wifi name
 String password = ""; // "thisisgreat"; // WiFi password
 
+// Our IP address in AP mode
```

Comment:
pl drop. I have two reasons, it breaks up the line to the "for TCP, UDP" section and one can piece it together only if one konws that these are AP, and it's not relevant that it's in AP mode